pkgs/applications/window-managers/sway/wrapper.nix

```diff
@@ -30,15 +30,17 @@ in symlinkJoin {
   paths = (optional withBaseWrapper baseWrapper)
     ++ [ sway-unwrapped ];
 
-  nativeBuildInputs = [ makeWrapper ]
-    ++ (optional withGtkWrapper wrapGAppsHook);
```

Comment:
What's the reason to remove `nativeBuildInputs`? Did that really break something? Because technically that's where `makeWrapper` and `wrapGAppsHook` belong (IIRC this currently doesn't help much for `symlinkJoin`, but there could be breaking changes in the future).

Reply:
I've tries a lot of combination to make this works, I'll try to revert this part.